Framework/Reflectometry/src/ReflectometryBackgroundSubtraction.cpp

```diff
@@ -290,7 +290,13 @@ std::map<std::string, std::string> ReflectometryBackgroundSubtraction::validateI
 
   MatrixWorkspace_const_sptr inputWS = getProperty("InputWorkspace");
   auto indexProp = dynamic_cast<IndexProperty *>(getPointerToProperty("ProcessingInstructions"));
-  Indexing::SpectrumIndexSet indexSet = *indexProp;
+  Indexing::SpectrumIndexSet indexSet;
+  try {
+    indexSet = *indexProp;
+  } catch (std::runtime_error &e) {
```

Comment:
I think this can be const
```suggestion
  } catch (const std::runtime_error &e) {
```